packages/experience/src/pages/MfaBinding/TotpBinding/index.module.scss

```diff
@@ -0,0 +1,7 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  > *:not(:first-child) {
+    margin-top: _.unit(6);
+  }
```

Comment:
Nit: consider using flex and gap instead

Reply:
Updated.